templates/zerver/emails/email.css

```diff
@@ -267,11 +258,6 @@ p.digest_paragraph,
 }
 
 @media only screen and (max-width: 620px) {
-    table[class="body"] h1 {
-        font-size: 28px !important;
-        margin-bottom: 10px !important;
-    }
```

Comment:
I can't prove this CSS doesn't do something -- we do support `h1` tags in missed-message emails, but I've confirmed the rest are not operative.